test/views/btn/CheckYourAnswersViewSpec.scala

```diff
@@ -62,8 +62,9 @@ class CheckYourAnswersViewSpec extends ViewSpecBase {
         "If you submit a Below-Threshold Notification for a previous accounting period, any return you have submitted this accounting period will be removed."
     }
 
-    "have a paragraph with a H3 heading" in {
-      view().getElementsByTag("h3").get(0).text mustEqual "Submit your Below-Threshold Notification"
+    "have a paragraph with a H2 heading" in {
+      view().getElementsByTag("h2").get(0).text mustEqual "Submit your Below-Threshold Notification"
+      view().getElementsByTag("h2").get(0).hasClass("govuk-heading-s") mustEqual true
```

Comment:
```suggestion
      val h2Headings: Elements = groupView.getElementsByTag("h2")
      h2Headings.get(0).text mustBe "Submit your Below-Threshold Notification"
      h2Headings.get(0).hasClass("govuk-heading-s") mustBe true
```

We should be using a consistent syntax in matchers e.g using `mustBe`, `shouldBe` or `mustEqual`

Reply:
I'm a little confused with this one - I thought I was doing this with mustEqual as that seemed to be the one being used but your suggestion is mustBe